test/maybe/check.cc

```diff
@@ -6,18 +7,63 @@ using namespace Napi;
 namespace {
 
 void VoidCallback(const CallbackInfo& info) {
-  Function fn = info[0].As<Function>();
+  Napi::Function fn = info[0].As<Function>();
+  Maybe<Value> fn_m = fn.Call({});
```

Comment:
This variable represents the return value of the fn invocation. Might be worth to name it with prefix like `ret`, rather than `fn`.

```suggestion
  Maybe<Value> ret_m = fn.Call({});
```

Reply:
Updated the PR to fix this.